plugins/modules/files/archive.py

```diff
@@ -242,6 +307,8 @@ def main():
     changed = False
     state = 'absent'
 
+    exclusion_patterns = params['exclusion_patterns'] if params['exclusion_patterns'] is not None else []
```

Comment:
```suggestion
    exclusion_patterns = params['exclusion_patterns'] or []
```

Reply:
You marked this as resolved without any comment, and without any change. Please either comment on why you don't want to do this, or apply it somehow :)

Reply:
Missed it on first glance. Applied with the latest comment